### ISSUE TITLE
Agree on the released name.

### ISSUE THREAD

**Issue by yakutovicha** [Probe-Particle/ProbeParticleModel#31]

**Comment by NikoOinonen** [comment]
I think @ondrejkrejci was suggesting in the meeting `ppafm`, which I like. It's short and to the point.

**Comment by yakutovicha** [comment]
> I think @ondrejkrejci was suggesting in the meeting `ppafm`, which I like. It's short and to the point.

I agree. Anyone else has different ideas? @ProkopHapala, @mondracek?

**Comment by yakutovicha** [comment]
Ok, if there are no other opinions, we stick with @ondrejkrejci's suggestion to call it `ppafm`.
